apps/ide/src/plugins/webida.preference/plugin.js

```diff
@@ -28,6 +28,7 @@ define([
     'webida-lib/util/logger/logger-client',
     'webida-lib/plugins/workspace/plugin',
     'webida-lib/util/path',
+    'webida-lib/plugins/workbench/ui/promiseMap',
```

Comment:
please move up the line 'webida-lib/plugins/workbench/ui/promiseMap', one step above.